Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -0,0 +1,415 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class ProjectPageViewController: UIViewController, MessageBannerViewControllerPresenting {
+  // MARK: Properties
+
+  private enum NavigationButtonSizes: CGFloat {
+    case spacing = 15.0
+  }
+
+  private let viewModel: ProjectPageViewModelType = ProjectPageViewModel()
+  private let shareViewModel: ShareViewModelType = ShareViewModel()
+  private let watchProjectViewModel: WatchProjectViewModelType = WatchProjectViewModel()
+  /**
+   FIXME: This `contentController` can be renamed `contentViewController` and has to be embedded in a `PagingViewController` in https://kickstarter.atlassian.net/browse/NTV-195
+   Maybe check `BackerDashboardViewController`'s pageViewController for in-app examples on how to do this.
+   */
+  private var contentController: ProjectPamphletContentViewController?
```

Comment:
yeah let's name this `contentViewController` so it matches the object it's holding the value of.